README.md

```diff
@@ -16,6 +16,16 @@ ember new my-app -b @ember/octane-app-blueprint
 ember addon my-addon -b @ember/octane-addon-blueprint
 ```
 
+Existing Projects
+
+```bash
+# Apps
+ember init -b @ember/octane-app-blueprint
+
+# Addons
+???
```

Comment:
I actually think its the same command (but with a different `-b` option)

Reply:
cool